src/set_theory/surreal.lean

```diff
@@ -352,10 +352,23 @@ instance : add_semigroup surreal :=
 { add_assoc := add_assoc,
   ..(by apply_instance : has_add surreal) }
 
+theorem zero_add : ∀ (x : surreal), 0 + x = x :=
+by rintro ⟨x, ox⟩; exact quotient.sound (pgame.zero_add_equiv _)
```

Comment:
```suggestion
by { rintro ⟨x, ox⟩,  exact quotient.sound (pgame.zero_add_equiv _) }
```
The style guide asks to not use `;` except when there are actually multiple goals being passed between the tactics.

Reply:
Fixed it. Sorry about that.